pymllm/orchestrator/tokenizer_process.py

```
@@ -0,0 +1,108 @@
+            self._send_to_scheduler.send_pyobj(tokenized)
+
+    # ------------------------------------------------------------------
+    # Tokenization (placeholder)
+    # ------------------------------------------------------------------
+
+    def _tokenize(self, raw_request: Dict[str, Any]) -> Dict[str, Any]:
+        """Tokenize a single raw request and return the tokenized payload.
+
+        TODO: replace with real tokenizer call.
+        """
+        text = raw_request.get("text", "")
```

Ruff (0.15.1)

[error] Local variable `text` is assigned to but never used

Remove assignment to unused variable `text`

(F841)